src/hangedMan/domain/AbstractGallows.java

```diff
@@ -1,3 +1,5 @@
+package hangedMan.domain;
```

Comment:
thw word "domain" is usually used for database, or other data model/storage related usage. In this case, you have put a lot of different classes in domain package which all serve a different purpose.

Reply:
Ok, thank you for the info. I guess I find out with bigger programs.